src/Moq/Obsolete/Mock.Legacy.cs

```diff
@@ -1,24 +0,0 @@
-// Copyright (c) 2007, Clarius Consulting, Manas Technology Solutions, InSTEDD.
-// All rights reserved. Licensed under the BSD 3-Clause License; see License.txt.
-
-using System;
-using System.Linq.Expressions;
-
-using Moq.Matchers;
-
-namespace Moq
-{
-	// Keeps legacy implementations.
-	public partial class Mock
-	{
-		[Obsolete]
-		internal static void VerifySet(Mock mock, LambdaExpression expression, Times times, string failMessage)
```

Comment:
Please check once more whether it is really safe to delete this. Which other methods used this one? What are they calling now instead?